lib/deliveries_manager/lib/deliveries_manager.rb

```diff
@@ -3,5 +3,24 @@
 require 'deliveries_manager/engine'
 
 module DeliveriesManager
-  # Your code goes here...
+  def self.processPayload(payload)
```

Comment:
have you used sidekiq in this project - this can be a good candidate to try it out

Reply:
I learned about it, but couldn't figure out how it can be used in this exercise. Because of the blocking nature of the subscribe method, I thought it can be a good idea to use it in a containerised rake task.